### ISSUE TITLE
Clarification on “sharding” mentioned in section 5

### ISSUE THREAD

**Issue by bmcase** [wangshan/draft-wang-ppm-differential-privacy#40]
In Section 5 it mentions 
> An optional Aggregator randomization mechanism that adds noise to an Aggregator's aggregate share prior to unsharding.

Is "sharding" here talking multiple servers run by the Aggregator parties processing different parts of the data and adding noise to each?   Because that will be more noise than necessary and we should think about adding noise after the unsharding but before revealing the results.  I may be miss understanding the terminology distinguishing "Aggregator" which I assume to be a Party that could be processing multiple "shards" of data on many servers.

**Comment by junyechen1996** [comment]
> Is "sharding" here talking multiple servers run by the Aggregator parties processing different parts of the data and adding noise to each?

You are right about "sharding" (and "unsharding" I assume), and one Aggregator processes one piece of data (i.e., "input share") from sharding. We've been referencing the terms in the VDAF draft for those ([sharding](https://www.ietf.org/archive/id/draft-irtf-cfrg-vdaf-07.html#name-sharding-2) and [unsharding](https://www.ietf.org/archive/id/draft-irtf-cfrg-vdaf-07.html#name-unsharding-2)). 

> Because that will be more noise than necessary and we should think about adding noise after the unsharding but before revealing the results.

That is true that more noise will be added than necessary. Usually the "collector" is the party that sees the aggregate result in the clear, so it will be too late to add DP noise then, i.e., the exact result have already been revealed by then.
